app/steps/ui/will/neworiginal/index.js

```diff
@@ -0,0 +1,22 @@
+const ValidationStep = require('app/core/steps/ValidationStep'),
+    json = require('app/resources/en/translation/will/original.json');
+
+module.exports = class NewWillOriginal extends ValidationStep {
```

Comment:
Please export the module at the bottom of the file, see https://github.com/hmcts/probate-tech-docs/blob/master/javascript-coding-style.md#module-exports for an example